exp/urls.py

```diff
@@ -41,5 +42,7 @@
     url(r'studies/create/$', StudyCreateView.as_view(), name='study-create'),
     url(r'studies/(?P<pk>\d+)/$', StudyDetailView.as_view(), name='study-detail'),
     url(r'studies/(?P<pk>\d+)/edit/$', StudyUpdateView.as_view(), name='study-edit'),
+    url(r'studies/(?P<pk>\d+)/edit/build/$', StudyBuildView.as_view(), name='study-build'),
+    url(r'^studies/(?P<study_uuid>[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89ab][0-9a-fA-F]{3}-[0-9a-fA-F]{12})/preview/?$', PreviewProxyView.as_view(), name='preview-proxy'),
```

Comment:
The `ProxyView` wants a path kwarg in the URL. If you look at the [one for taking the experiment](https://github.com/CenterForOpenScience/lookit-api/blob/develop/web/urls.py#L16) it wraps the entire url after `/studies/` in a named capture group called path. I think the other two kwargs, study_uuid and child_uuid are also available but I'm not sure.

Also, I'm not sure that you need the `^`. That indicates the beginning of a line and I can't remember if django's URL inheritance fixes that for you or not, just that the others above this one are working without it.

Make sense? If adding path in here gives you trouble let me know and I'll pull this down and step through it with ipdb.